jbmc/src/janalyzer/janalyzer_parse_options.cpp

```diff
@@ -632,7 +632,7 @@ bool janalyzer_parse_optionst::process_goto_program(const optionst &options)
     goto_partial_inline(goto_model, ui_message_handler);
 
     // remove returns, gcc vectors, complex
-    remove_returns(goto_model);
+    remove_returns(log.get_message_handler(), goto_model);
```

Comment:
I'm not sure we're entirely consistent with this, but I think we mostly pass a message handler as one of the later parameters. So here I'd swap the two.

Reply:
Well at least for `remove_function_pointers` the `message_handler` is the first parameter. But ok, swapped.